client-v2/src/components/FrontsEdit/Front.tsx

```diff
@@ -134,6 +136,16 @@ class FrontComponent extends React.Component<FrontProps, FrontState> {
     }
   }
 
+  public addImageToArticleFragment(
```

Comment:
Is this a duplicate of the one function which exists in the clipboard component? Could we add to some utilities file perhaps with the `getImageMetaFromValidationResponse` function?

Reply:
Done 🆗